src/components/cards/GalleryCard.jsx

```diff
@@ -20,7 +20,7 @@ export default function GalleryCard({
     <Card
       title={title}
       titleId={titleId}
-      renderActions={<Link>See all</Link>}
+      renderActions={<Link href="temp.url">See all</Link>}
```

Comment:
this components are not actually used, so for now i'm just making eslint shut up. soon we need to evaluate whether this component should be thrown out or if there is some valuable work to salvage.